lib/chef_backup/strategy/backup/tar.rb

```diff
@@ -122,7 +122,6 @@ def write_manifest
 
   DEFAULT_STATEFUL_SERVICES = %w[rabbitmq
                                  opscode-solr4
-                                 elasticsearch
```

Comment:
Chef Infra Server also uses chef_backup to take the server backup. Removing elasticsearch from this list would imply that elasticsearch backup cannot be taken for chef-server as well. This would break the current user experience.

It would be nice if we can make the change to include /exclude elasticsearch in backups based on a configuration or as an cli option.

Reply:
But I think creating  a cli option can be done as a part of a separate ticket, I do not want to hold this change up for that to happen. 
An alternative could be that backup is taken only when
1. Service is stateful
2. Is enabled.
So disabling the elasticsearch service in chef-backend before populating_data_map should remove elasticsearch from the  stateful services. And we would no longer see elasticsearch data bieng backedup only for chef-backend.

That would look like:
https://github.com/chef/chef-backend/blob/master/libcb/lib/libcb/command/backup.rb#L82 before
https://github.com/chef/chef-backend/blob/master/libcb/lib/libcb/command/backup.rb#L74 and there would be no changes required to chef_backup.